src/schema/bookmarks.ts

```diff
@@ -300,22 +307,39 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
       source,
       { data }: { data: GQLAddBookmarkInput },
       ctx: AuthContext,
-    ): Promise<GQLEmptyResponse> => {
+      info,
+    ): Promise<GQLBookmark[]> => {
+      const lastAddedBookmark = await ctx.con.getRepository(Bookmark).findOne({
+        where: { userId: ctx.userId },
+        select: ['listId'],
+      });
+      const lastUsedListId = lastAddedBookmark?.listId ?? null;
+
       const [query, params] = ctx.con
         .createQueryBuilder()
         .select('id', 'postId')
         .addSelect(`'${ctx.userId}'`, 'userId')
+        .addSelect(`:listId`, 'listId')
```

Comment:
Why the weird select setting?
```suggestion
        .addSelect('listId')
```

Reply:
Updated ✔️